include/datetime.php

```diff
@@ -180,10 +180,6 @@ function datetime_convert($from = 'UTC', $to = 'UTC', $s = 'now', $fmt = "Y-m-d
 function dob($dob) {
 	list($year,$month,$day) = sscanf($dob,'%4d-%2d-%2d');
 
-	$f = Config::get('system', 'birthday_input_format');
```

Comment:
Please remove the documentation for this config value from "doc/htconfig.md" as well.